static/app/components/feedback/feedbackItem/feedbackItemLoader.tsx

```diff
@@ -22,10 +22,13 @@ export default function FeedbackItemLoader() {
   useSentryAppComponentsData({projectId: projectSlug});
 
   useEffect(() => {
+    if (issueResult.isError) {
+      trackAnalytics('feedback.feedback-item-render-error', {organization, feedbackId});
+    }
```

Comment:
Is there no sentry error when this happens? We should not need to capture error analytics :)

Reply:
hmm I don't see anything in Issues....

Reply:
Here's a replay of clicking into a feedback that errors: https://sentry.sentry.io/replays/abd88bfa16114ef5b682f88d203b223a/?n_detail_row=341&project=11276&query=user.email%3Acatherine.lee%40sentry.io&referrer=%2Freplays%2F%3AreplaySlug%2F&statsPeriod=90d&t_main=network&yAxis=count%28%29&t=140 no Sentry error tho

Reply:
it seems like there was no error because of the `issueData?` down below: it's possible that the data doesn't exist and we handle it without throwing. Not an error, but also it wasn't a good user experience until now.

Tracking how often are people asking for feedbacks that don't exist seems reasonable

Reply:
How about `feedback.feedback-item-missing-data`?

Reply:
I think I did something similar for feedback trace data, I also called the event an "error"..

Reply:
Nit: can split these if conditions to separate useEffect's